Source/warningreporting/WarningReportingUnit.cpp

```diff
@@ -113,112 +106,72 @@ namespace WarningReporting {
         ASSERT(doorBell.empty() == false);
 
         if (fileName.empty() == false) {
-       
-            fileName +=  '.' + Core::NumberType<uint32_t>(identifier).Text();
+
+            fileName += '.' + Core::NumberType<uint32_t>(identifier).Text();
```

Comment:
Just to make sure diud we check if this now doesn't colide with tracing? (I perhaps just copied the tracing code)